lzy/execution-env/src/main/java/ai/lzy/env/logs/LogStreamCollection.java

```diff
@@ -0,0 +1,27 @@
+package ai.lzy.env.logs;
+
+import jakarta.annotation.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+public class LogStreamCollection {
+    private final List<LogStream> streams = new ArrayList<>();
+
+    public LogStream stream(String name) {
+        var stream =  new LogStream(name, null);
```

Comment:
it'd be better to replace this function body with simple forward call `return stream(name, null);`